platform/ABModelApiNetsuite.js

```diff
@@ -15,9 +15,159 @@ const CONCURRENCY_LIMIT = 20;
 // {int}
 // This is the number of parallel operations we want to limit ourselves to
 // so we avoid trippig NetSuit's CONCURRENCY_LIMIT_EXCEEDED
+// This is local to our Relate/unRelate operations
+
+const CONCURRENCY_LIMIT_MAX = 200;
```

Comment:
Should this be configurable, maybe through an env  variable?

Reply:
I agree.  After I posted this, I spent the rest of the weekend thinking ... "I should have made that an .env variable..."